src/Index/page.jsx

```diff
@@ -98,6 +98,11 @@ class IndexPageComponent extends React.Component {
                 {homepageCharts}
               </Carousel>
             </MediaQuery>
+            {homepageEmailForm
```

Comment:
you probably want to do this for the `nctIndex` page instead

NCT is on a special portal bundle

Reply:
thanks for commenting i was trying to get this to show and wondering why nothing was working